### DISCUSSION TITLE
on the possibility of cell network outages in europe

### DISCUSSION THREAD

**Discussion by trosel** [markqvist/Reticulum#106] [General]
just thought this may be relevant to reticulum

https://www.zerohedge.com/technology/gone-30-minutes-next-europes-doomsday-list-collapse-cell-phone-networks

**Comment by markqvist** [comment]
The 30 minute figure is not an overstatement or just dramatic effect. On all of the cell sites and general-purpose antenna towers I have seen or worked on, only a very small number of them had adequate backup power for even that.

Most of the tower sites had battery backups installed when the site was commissioned, with no subsequent maintenance or upgrades to capacity when more power-hungry equipment was added.

It is going to be impossible for grid operators to "isolate cell towers" and keep them on "priority power" as the article mentions. The electricity grid is not built that way, and due to the widely dispersed nature of the cell-sites, they are almost always just connected to normal residential power circuits, fed by the same local transformer station. Local transformer-level control is going to be the maximum level of granularity the grid operators have almost everywhere, which of course means that if the power is dropped, 95% of cell sites drop too within a *very* short amount of time.

Thanks for sharing this, it was an interesting read.

**Reply by trosel** [reply]
could be a good blog idea. How reticulum could help in such a situation

**Reply by markqvist** [reply]
Point taken! Thanks for the idea!
